data/localization/translate.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Translation Helper Tool | EmulatorJS</title>
+    <link rel="icon" type="image/png" href="../../docs/favicon.ico" />
+</head>
+
+<body>
+    <div class="header">
+        <h1>Translation Helper Tool</h1>
+        <img style="width: 150px" src="../../docs/Logo.png" />
+    </div>
+    <br />
+    <div class="boxes">
+        <div class="box">
+            <button class="copyBtn" id="copyBtn1" onclick="copy('box1')">
```

Comment:
Javascript shouldnt be called inline, you should add an event listener in the javascript for the button id